doc/source/serve/advanced.rst

```diff
@@ -269,3 +269,35 @@ Once a endpoint is deleted, its tag can be reused.
 .. code-block:: python
 
   serve.delete_endpoint("simple_endpoint")
+
+How do I get a reference to a "backend" class?
```

Comment:
"reference" isn't very clear to me - what about "how to I call an endpoint from within my backend?"